deploy/README.md

```diff
@@ -108,6 +108,11 @@ nginx-ingress-controller-fdcdcd6dd-vvpgs   1/1       Running   0          11s
 ### AWS
 
 In AWS we use an Elastic Load Balancer (ELB) to expose the NGINX Ingress controller behind a Service of `Type=LoadBalancer`.
+Since Kubernetes v1.9.0 it is possible to use a classic load balancer (ELB) or 
```

Comment:
should be `(ELB) or network load balancer (NLB)`

Reply:
@Rishka thanks :)